the-collab-lab/tcl-35-smart-shopping-list
# Aa fo create prompt btn

## Description

This PR created a welcoming prompt to add a new user's first item if the list is empty to help the user gets oriented to how the app works.
The code changes the way the new user navigates to add item page for UX purposes.

@fatima-ola & @adiati98 learned how to route to a page on click event, which is using the `useHistory().push` method from `react-router-dom`.



## Related Issue

Closes #7 



## Acceptance Criteria

- The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After


![add-item-btn](https://user-images.githubusercontent.com/45172775/139330015-7fdeddca-fc23-439f-9e7e-30508d19ba06.jpg)



## Testing Steps / QA Criteria

- Pull the branch.
- Run `npm start`
- Launch the browser to preview at `http://localhost:3000/`
- Click the "Create a new list" button.
- The screenshot above appears, and the new user can add an item by clicking the "Add item" button.
